.env.unfcu.example

```
@@ -0,0 +1,22 @@
+# UNFCU Edition Environment Variables
+# Copy to .env and fill in real values before deployment.
+# Generate JWT secret with: openssl rand -base64 64
+
+# Required: JWT authentication secret (must match Supabase project secret)
+SUPABASE_JWT_SECRET=REPLACE_ME_WITH_64_CHAR_SECRET
+JWT_SECRET=REPLACE_ME_WITH_64_CHAR_SECRET
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the SUPABASE_JWT_SECRET key

(UnorderedKey)